### PR TITLE
Updating Toshi homepage contents and statistics

### LINKED ISSUE #1: [web/api] update homepage to show Toshi logo and info
- Update web assets (icons, logo, etc.)
- Update default placeholder (search bar, etc.)
- Update homepage info (features, stats, etc.)

### PR DESCRIPTION
**Pull request by sajithaliyanage**
Closes #1

### CODE REVIEW

File: web/public/index.html [hunk 1]
```diff
@@ -6,26 +6,26 @@
...
-    <meta name="theme-color" content="#1976d2" />
+    <meta name="theme-color" content="#1976D2" />
     <meta
       name="description"
-      content="The most advanced search engine for the darkweb"
+      content="The most advanced cryptocurrency wallet explorer"
     />
     <meta name="author" content="Qatar Computing Research Institute" />
-    <meta name="keywords" content="Advanced,Search,Engine,Dark,Web" />
+    <meta name="keywords" content="Advanced,Search,Engine,Explorer," />
```
Comment (boshmaf) [inline web/public/index.html:15]: Cryptocurrency,Wallet inplace of Search,Engine (also remove the last comma)

File: web/public/index.html [hunk 2]
```diff
@@ -6,26 +6,26 @@
...
 
     <meta property="og:locale" content="en_US" />
     <meta property="og:type" content="website" />
-    <meta property="og:title" content="Dizzy" />
+    <meta property="og:title" content="Toshi" />
     <meta property="og:image" content="%PUBLIC_URL%/dizzy-logo-white.png" />
```
Comment (boshmaf) [inline web/public/index.html:20]: toshi-logo-white.png not dizzy-logo-white.png

File: web/public/index.html [hunk 3]
```diff
@@ -6,26 +6,26 @@
...
     <meta
       property="og:description"
-      content="The most advanced search engine for the darkweb"
+      content="The most advanced cryptocurrency wallet explorer"
     />
     <meta property="og:url" content="https://dizzy.cibr.qcri.org" />
```
Comment (boshmaf) [inline web/public/index.html:25]: https://toshi.cibr.qcri.org

File: web/public/index.html [hunk 4]
```diff
@@ -6,26 +6,26 @@
...
-    <meta property="og:site_name" content="Dizzy" />
+    <meta property="og:site_name" content="Toshi" />
 
-    <link rel="icon" href="%PUBLIC_URL%/dizzy-logo-transparent.png" />
+    <link rel="icon" href="%PUBLIC_URL%/toshi-logo-transparent.png" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/dizzy-logo-transparent.png" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/dizzy-logo-white.png" />
```
Comment (boshmaf) [inline web/public/index.html:30]: toshi not dizzy

File: web/src/components/LogoRaw/LogoRaw.js [hunk 5]
```diff
@@ -21,7 +18,7 @@ const LogoRaw = (props) => {
   const view = (
     <div className={classes.root}>
       <Link className={classes.link} component={NavLink} to="/main">
-        <img src={logo} alt="Dizzy" height={height} />
+        <h1 className="MuiTypography-root MuiTypography-h1">Toshi</h1>
```
Comment (boshmaf) [inline web/src/components/LogoRaw/LogoRaw.js:21]: Please use images not text.

File: web/src/components/Main/Stats/Stats.js [hunk 6]
```diff
@@ -18,28 +18,35 @@ const MainStats = () => {
   const classes = useStyles();
   const dispatch = useDispatch();
   const stats = useSelector((state) => state.stats.data.computed);
-  const cryptosCount = Object.values(stats.count.crypto).reduce(
-    (a, b) => a + b
-  );
+  const [riskLevel, setRiskLevel] = useState("N/A");
```
Comment (boshmaf) [inline web/src/components/Main/Stats/Stats.js:21]: Remove plz

File: web/src/components/SearchBoxRaw/SearchBoxRaw-styles.js [hunk 7]
```diff
@@ -2,7 +2,7 @@ export const stylesCreator = (theme) => ({
   Default: {
     root: {
       width: '100%',
-      maxWidth: 460,
```
Comment (boshmaf) [inline web/src/components/SearchBoxRaw/SearchBoxRaw-styles.js:5]: Please return previous value

File: web/src/components/SearchBoxRaw/SearchBoxRaw.js [hunk 8]
```diff
@@ -23,7 +23,7 @@ const SearchBoxRaw = (props) => {
   // Variables
   const {
     classes,
-    placeholder = 'Search the darkweb',
+    placeholder = 'Search your wallet by address, transaction or label',
```
Comment (boshmaf) [inline web/src/components/SearchBoxRaw/SearchBoxRaw.js:26]: "Search Bitcoin wallets" plz

File: web/src/components/StatRaw/StatRaw.js [hunk 9]
```diff
@@ -2,7 +2,7 @@
 import React, { useEffect, useState } from 'react';
 
 // PropTypes
-import PropTypes from 'prop-types';
+import PropTypes, {oneOfType} from 'prop-types';
```
Comment (boshmaf) [inline web/src/components/StatRaw/StatRaw.js:5]: Remove plz and everything related to it

File: web/src/store/reducers/stats.js [hunk 10]
```diff
@@ -9,11 +9,9 @@ const initialState = {
...
     computed: {
       count: {
-        page: 0,
-        domain: 0,
-        crypto: {
-          btc: 0,
-        },
+        wallet: 0,
+        label: 0,
+        riskLevelScore: -1,
```
Comment (boshmaf) [inline web/src/store/reducers/stats.js:14]: no risk plz

**Review by boshmaf — CHANGES_REQUESTED** [review]

File: api/controllers/statistics.js [hunk 11]
```diff
@@ -1,30 +1,30 @@
...
-    return next(new ErrorResponse('There are no statistics available', 404));
+    if (!statistics) {
+      return next(new ErrorResponse('There are no statistics available', 404));
+    }
   }
 
   return response.status(200).json({
     success: true,
-    data: statistics,
+    data: statistics[0],
```
[the remark this answers is not in the document]
Comment (sajithaliyanage) [reply api/controllers/statistics.js]: Fixed

File: api/jobs/computeStats.js [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+  try {
+    await Statistic.deleteMany({});
+  } catch (error) {
+    console.log(`[cron:computeStats] Error while deleting previous record`.red);
+    console.log(error);
+  }
+
+  const count = {};
+  try {
+    const walletRequest = gp.query('SELECT count(*) from btc_wallet;');
```
[the remark this answers is not in the document]
Comment (sajithaliyanage) [reply api/jobs/computeStats.js]: Fixed

File: api/jobs/computeStats.js [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+    const labelRequest = gp.query('SELECT count(distinct label) from btc_address_label;');
```
[the remark this answers is not in the document]
Comment (sajithaliyanage) [reply api/jobs/computeStats.js]: Fixed

File: api/jobs/computeStats.js [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+    const riskLevelRequest = gp.query('SELECT avg(risk_score) from btc_wallet;');
+    const [walletResponse, labelResponse, riskLevelResponse] = await Promise.all([walletRequest, labelRequest, riskLevelRequest]);
```
[the remark this answers is not in the document]
Comment (sajithaliyanage) [reply api/jobs/computeStats.js]: Fixed

File: api/jobs/index.js [hunk 15]
```diff
@@ -19,9 +21,10 @@ const scheduleJobs = () => {
     if (process.env.PM2_INSTANCE_ID === '0') {
       cron.schedule('0 0 * * *', deleteInactiveUsers);
       cron.schedule('0 0 * * *', sendAlerts);
+      cron.schedule('0 0 * * *', computeStats);
```
[the remark this answers is not in the document]
Comment (sajithaliyanage) [reply api/jobs/index.js]: Fixed

File: api/models/Statistic.js [hunk 16]
```diff
@@ -6,14 +6,6 @@ const StatisticSchema = new mongoose.Schema(
...
     },
-    type: {
-      type: String,
-      required: true,
-      enum: ['index', 'domain'],
-    },
-  },
-  {
-    timestamps: true,
   }
```
[the remark this answers is not in the document]
Comment (sajithaliyanage) [reply api/models/Statistic.js]: Fixed

File: web/public/index.html (hunk 1 above)
Comment (sajithaliyanage) [reply web/public/index.html]: Fixed

File: web/public/index.html (hunk 2 above)
Comment (sajithaliyanage) [reply web/public/index.html]: Fixed

File: web/public/index.html (hunk 3 above)
Comment (sajithaliyanage) [reply web/public/index.html]: Fixed

File: web/public/index.html (hunk 4 above)
Comment (sajithaliyanage) [reply web/public/index.html]: Fixed

File: web/src/components/LogoRaw/LogoRaw.js (hunk 5 above)
Comment (sajithaliyanage) [reply web/src/components/LogoRaw/LogoRaw.js]: Fixed

File: web/src/components/Main/Stats/Stats.js (hunk 6 above)
Comment (sajithaliyanage) [reply web/src/components/Main/Stats/Stats.js]: Fixed

File: web/src/components/Main/Stats/Stats.js [hunk 17]
```diff
@@ -18,28 +18,35 @@ const MainStats = () => {
...
+  useEffect(() => {
+    if (stats.count.riskLevelScore >= 0.75) {
+        setRiskLevel("HIGH")
+    } else if (stats.count.riskLevelScore < 0.75 && stats.count.riskLevelScore >= 0.5) {
+        setRiskLevel("MEDIUM")
+    } else if (stats.count.riskLevelScore < 0.5 && stats.count.riskLevelScore >= 0) {
+        setRiskLevel("LOW")
+    }
+  }, [stats]);
+
```
[the remark this answers is not in the document]
Comment (sajithaliyanage) [reply web/src/components/Main/Stats/Stats.js]: Fixed

File: web/src/components/SearchBoxRaw/SearchBoxRaw-styles.js (hunk 7 above)
Comment (sajithaliyanage) [reply web/src/components/SearchBoxRaw/SearchBoxRaw-styles.js]: Fixed

File: web/src/components/SearchBoxRaw/SearchBoxRaw.js (hunk 8 above)
Comment (sajithaliyanage) [reply web/src/components/SearchBoxRaw/SearchBoxRaw.js]: Fixed

File: web/src/components/StatRaw/StatRaw.js (hunk 9 above)
Comment (sajithaliyanage) [reply web/src/components/StatRaw/StatRaw.js]: Fixed

File: web/src/store/reducers/stats.js (hunk 10 above)
Comment (sajithaliyanage) [reply web/src/store/reducers/stats.js]: Fixed

**Pull request merged**